src/components/Tooltip/index.js

```diff
@@ -149,6 +150,7 @@ class Tooltip extends PureComponent {
         Animated.timing(this.animation, {
             toValue: 0,
             duration: 140,
+            useNativeDriver: false,
```

Comment:
Why not use UI thread for these animations?

Reply:
Yes, but no. We just weren't using it before (default is false) and the main thing that has changed is that _something_ now complains that we **must** specify a value in the console so I explicitly added `false` here to preserve what we had before and silence the warning.

Reply:
Okay cool, lets leave this for now then. It might be worth looking into this separately, so I created a [monthly issue here](https://github.com/Expensify/Expensify/issues/166695).